include/wx/generic/combo.h

```diff
@@ -92,7 +92,7 @@ class WXDLLIMPEXP_CORE wxGenericComboCtrl
     virtual GtkEntry *GetEntry() const wxOVERRIDE { return NULL; }
 #endif
 #elif defined(__WXMAC__)
```

Comment:
This is trivial, but it just rubs me the wrong way that we use `__WXMAC__` here around the declaration and `__WXOSX__` in the source file around the implementation. Could we standardize on one of them (the latter is preferred, I think?) in both places? Or maybe just move the implementation inline to avoid the problem :-/

Reply:
yes __WXOSX__ is definitely the preferred, I didn't touch the constant in the header file, but I'll move everything inline, you are right